server/src/user/service.js

```diff
@@ -1,10 +1,15 @@
 const UserModel = require('./model')
 const bcrypt = require('bcryptjs')
 const errorHandler = require('../util/errorUtil')
+const mailer = require('../services/mailer')
+const crypto = require('crypto')
 
 exports.add = (req, res) => {
+  const token = crypto.createHash('sha1').update(Date.now().toString() + Math.random().toString()).digest('hex')
+  console.log(token)
   let newUser = new UserModel({
     email: req.body.email,
+    token: token,
```

Comment:
Remove the console log and the token const before and use here:
```
token: crypto.createHash('sha1').update(Date.now().toString() + Math.random().toString()).digest('hex')
or
token: crypto.createHash('sha1').update(Date.now() + '' + Math.random()).digest('hex')
```

Reply:
done